src/core/main.lua

```diff
@@ -33,23 +33,21 @@ function main ()
    zone("startup")
    require "lib.lua.strict"
    initialize()
+   local program
```

Comment:
@capr I don't think the changes to `core/main` should be in this PR. Are they required for lisper to work?